### DISCUSSION TITLE
Resize modals

### DISCUSSION THREAD

**Discussion by hawwokitty** [React95/React95#459] [Q&A]
Hello! Is there a way to be able to adjust the size of windows? like the modals? by dragging the corner?

**Comment by ggdaltoso** [comment]
At this moment it is not possible since our Modal does not accept CSS classes to extend its behavior/styles.

It should be as simple as:

```jsx
// somewhere in your application
.resize {
  resize: both;
  overflow: auto;
}

<Modal className="resize" />
```


https://github.com/user-attachments/assets/c5b80f9b-c72a-4619-af3b-31f7970f9a7f



However, our API does not support such a feature.

I'm working to improve Modals and include the `className` extension.

Thanks for suggesting! I'm happy to know you're using and contributing to this library!

**Reply by hawwokitty** [reply]
Thanks! i ended up using the react-resizable node module as a temporary fix!
